pkg/apis/autoscaling/v1/clusterautoscaler_types.go

```diff
@@ -53,6 +64,16 @@ type ClusterAutoscalerSpec struct {
 	// This option has priority over log level set by the `CLUSTER_AUTOSCALER_VERBOSITY` environment variable.
 	// +kubebuilder:validation:Minimum=0
 	LogVerbosity *int32 `json:"logVerbosity,omitempty"`
+
+	// Sets the type and order of expanders to be used during scale out operations.
+	// This option specifies an ordered list, highest priority first, of expanders that
+	// will be used by the cluster autoscaler to select node groups for expansion
+	// when scaling out.
+	// If not specified, the default value is `random`, available options are: `least-waste`, `priority`, `random`.
```

Comment:
Would be nice to explain what each one of those options means, and also, why you can select multiple. In which scenario would the CA fallback to the second or third expander in the list

Reply:
i can add that